scipy/stats/contingency.py

```diff
@@ -13,17 +13,18 @@
    crosstab
    expected_freq
    margins
-
+   association
 """
 
 
 from functools import reduce
 import numpy as np
 from .stats import power_divergence
+import math
 from ._crosstab import crosstab
 
 
-__all__ = ['margins', 'expected_freq', 'chi2_contingency', 'crosstab']
+__all__ = ['margins', 'expected_freq', 'chi2_contingency', 'crosstab', 'association']
```

Comment:
[PEP-8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) line length:
```suggestion
__all__ = ['margins', 'expected_freq', 'chi2_contingency', 'crosstab',
           'association']
```